inclusions/slide-down-nav/help-help-section.html

```diff
@@ -0,0 +1,18 @@
+<h3>Help</h3>
+<img class="pull-image-right hidden-sm hidden-xs" src="images/drawers/placeholder.jpg" />
+<p class="emphasize-text">Account Resources</p>
+<ul class="drawer-link-list no-flex-lg">
+	<li>
+		<a href="#loan-periods"><i class="fas fa-user-circle btn-icon-left"></i> <span>Account Help</span></a>
+	</li>
+	<li>
+		<a href="#loan-periods"><i class="fas fa-book btn-icon-left"></i> <span>Catalog Help</span></a>
```

Comment:
Should every link be going to #loan-periods?

Reply:
@mxsnyder we can talk about this later, but this is only a placeholder for web services to update the link. If we want to actually source what is on the page (snippet) we can do that.